float32 thumb_target
float32 index_target
float32 middle_target

int8 thumb_vib_1
int8 thumb_vib_2
int8 thumb_vib_3

int8 index_vib_1
int8 index_vib_2
int8 index_vib_3

int8 middle_vib_1
int8 middle_vib_2
int8 middle_vib_3
